tests/integration/test_manifest_submission.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import os
+import shutil
+
+import pandas as pd
+import pytest
+import requests
+from synapseclient.client import Synapse
+
+from schematic.utils.general import create_temp_folder
+from tests.utils import CleanupItem
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="session")
+def download_location():
+    temporary_manifest_storage = "temporary_manifest_storage"
+    if not os.path.exists(temporary_manifest_storage):
+        os.makedirs(temporary_manifest_storage)
+    full_parent_path = os.path.abspath(temporary_manifest_storage)
+
+    download_location = create_temp_folder(full_parent_path)
+    yield download_location
+
+    # Cleanup after tests have used the temp folder
+    if os.path.exists(download_location):
+        shutil.rmtree(download_location)
+
+
+class TestManifestGeneration:
+    def test_submit_record_based_test_manifest_file_only(
+        self, helpers, syn_token, download_location, schedule_for_cleanup
+    ) -> None:
+        url = "http://localhost:3001/v1/model/submit"
+        data_type = "Biospecimen"
+        params = {
+            "schema_url": "https://raw.githubusercontent.com/Sage-Bionetworks/schematic/develop/tests/data/example.model.jsonld",
+            "data_model_labels": "class_label",
+            "data_type": data_type,
+            "dataset_id": "syn63561474",
+            "manifest_record_type": "file_only",
+            "restrict_rules": "false",
+            "hide_blanks": "false",
+            "asset_view": "syn63561606",
+            "table_manipulation": "replace",
+            "table_column_names": "class_label",
+            "annotation_keys": "class_label",
+            "file_annotations_upload": "false",
+        }
+
+        syn = Synapse()
+        syn.login(authToken=syn_token, silent=True)
```

Comment:
@linglp Should this go into conftest.py instead? Or come from the `synapse_store: SynapseStorage` instance that you can grab for the test?

Reply:
I could add this to conftest.py!